AppDB/backup/backup_recovery_helper.py

```diff
@@ -371,7 +371,7 @@ def deploy_apps(app_paths):
     appscale_info.get_secret())
 
   # Wait for Cassandra to come up after a restore.
-  time.sleep(5)
+  time.sleep(15)
```

Comment:
This should have something similar to what we do during boot: check when the port cassandra listen to is open. Not quite sure in python, but open the TCP socket should be enough to ensure the daemon is listening.